modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -45,6 +45,12 @@ In future releases, there is a possibility to deprecate the old method of custom
 :FeatureName: Argo CD Applications controller
 include::snippets/technology-preview.adoc[]
 
+* With this update, Argo CD supports the Server-Side Apply feature, which helps users to perform the following tasks:
+** Manage large resources which are too big to fit in the 262144 bytes allowed annotation size.
```

Comment:
"Manage large resources which are too big for the allowed annotation size of 262144 bytes." maybe?